src/ResourceManager/Websites/Commands.Websites/Cmdlets/DeploymentSlots/SetAzureWebAppSlot.cs

```diff
@@ -86,6 +86,10 @@ public class SetAzureWebAppSlotCmdlet : WebAppSlotBaseCmdlet
         [ValidateNotNullOrEmpty]
         public int NumberOfWorkers { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Enable or disable always on", ValueFromPipeline = true)]
```

Comment:
I don't think this is ValudFromPipeline.  If this is a property of the website object, then this should be ValueFromPipelineBtPropertyName